scripts/justinstall.sh

```diff
@@ -24,10 +24,10 @@ OPT_DIR=${INSTALL_DIR}/opt
 mkdir --mode=0755 -p ${OPT_DIR}
 cd ${OPT_DIR}
 
-mv ${WORKSPACE}/alti-livy/assembly/target/livy-server-${LIVY_VERSION}-incubating.zip ${OPT_DIR}/
+mv ${WORKSPACE}/incubator-livy/assembly/target/livy-${LIVY_VERSION}-incubating-SNAPSHOT-bin.zip ${OPT_DIR}/
```

Comment:
Does this impact the RPM name?

Reply:
No because RPM name is defined lower down as export RPM_NAME=`echo alti-livy-${LIVY_VERSION}`

Reply:
This was where the build put the zip file. And since it gets moved to another directory in line 27, and the mv target hasn't changed, no material change...

Reply:
RPM name turns out to be alti-livy-0.5.1-4.3.0-201807312206.x86_64.rpm